packages/boards/sender/platformio_upload.py

```
@@ -0,0 +1,143 @@
+        start_headers = {
+            'Host': host_ip,
+            'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/118.0',
+            'Accept': '*/*',
+            'Accept-Language': 'de,en-US;q=0.7,en;q=0.3',
+            'Accept-Encoding': 'gzip, deflate',
+            'Referer': f'{upload_url}/update',
+            'Connection': 'keep-alive'
+            }
+
+        try:
+            checkAuthResponse = requests.get(f"{upload_url_compatibility}/update")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)